go.mod

```
@@ -14,7 +14,7 @@ require (
 	github.com/prometheus-operator/prometheus-operator v0.53.1
 	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.53.1
 	github.com/prometheus-operator/prometheus-operator/pkg/client v0.53.1
-	github.com/prometheus/client_golang v1.11.0
+	github.com/prometheus/client_golang v1.11.1
```

we can bump to the latest stable release.
```suggestion
	github.com/prometheus/client_golang v1.12.1
```

---

Can we assume that every tag in [the Tag tab](https://github.com/prometheus/client_golang/tags) is stable release?

---

I find it ,the release list is not in an increasing order 😅 never mind